### PR TITLE
[HWORKS-210] jupyter threads fill up executor pool

### PR DESCRIPTION
**Pull request by o-alex**
https://hopsworks.atlassian.net/browse/HWORKS-210

### CODE REVIEW

File: recipes/install.rb [hunk 1]
```diff
@@ -267,6 +267,13 @@
               'maximumpoolsize' => 400,
               'taskqueuecapacity' => 20000,
               'description' => 'Hopsworks Conda Executor Service'
+          },
+          'concurrent/jupyterExecutorService' => {
```
Comment (SirOibaf) [inline recipes/install.rb:271]: This is problematic during the upgrade. When an upgrade runs this configuration doesn't get applied. You need to create the pool also in the default recipe.

File: recipes/install.rb [hunk 2]
```diff
@@ -267,6 +267,13 @@
...
+              'threadpriority' => 8,
+              'corepoolsize' => 300,
+              'maximumpoolsize' => 300,
+              'taskqueuecapacity' => 1000,
```
Comment (SirOibaf) [inline recipes/install.rb:275]: I think we would also benefit from making these parameters configurable.

**Review by SirOibaf — CHANGES_REQUESTED** [review]

File: recipes/install.rb (hunk 2 above)
Comment (SirOibaf) [reply recipes/install.rb:275]: Also I thought we didn't want to set the queue for this pool since it doesn't make much sense to wait

Comment (o-alex) [reply recipes/install.rb:275]: I will make the params configurable.
You need the queue to pause connections from tornado that do not have resources to run. If I put no queue, the connection will fail with an error, but tornado(jupyter) will eagerly try to reconnect... and we end up with reconnects every 100ms.

File: recipes/install.rb (hunk 1 above)
Comment (o-alex) [reply recipes/install.rb:271]: done

**Review by SirOibaf — APPROVED** [review]

**Pull request merged**